src/general/general.service.spec.ts

```diff
@@ -19,12 +19,12 @@ describe('GeneralService', () => {
           isGlobal: true,
         }),
         TypeOrmModule.forRoot({
-          type: process.env.geospatial_analyzer_db_type,
+          type: process.env.GEOSPATIAL_ANALYZER_DB_TYPE,
           host: 'localhost',
```

Comment:
If we do set this value in the `.env.test` (which makes sense for keeping config and code separate), we should use it:

```suggestion
          host: process.env.GEOSPATIAL_ANALYZER_DB_HOST,
```

Reply:
I've tried to implement all your suggestios to use wrapper functions/helpers instead of defining the database connection params and config params in each test seperately.
Helpers: b6e08a60edc1f9b7c873c2175fdb4eee0d61371f
Unit Test Integration: bdef144c7f195d573e57a8f68e8551f7689095cb
e2e Test Integration: eac5034fceaaaa243a89d068e7bede6253728da9

Reply:
Really nice! Thanks for the clean up!